react/components/Error.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import { useCssHandles } from 'vtex.css-handles'
+
+const CSS_HANDLES = ['defaultErrorContainer', 'defaultErrorMessage'] as const
+
+function DefaultError() {
+  const handles = useCssHandles(CSS_HANDLES)
+
+  return (
+    <div className={`flex justify-center pv5 ${handles.defaultErrorContainer}`}>
+      <p className={handles.defaultErrorMessage}>
```

Comment:
Can we just render a `p` element instead with no styling?

Reply:
Yep, we can. I guess it just looked very ugly and I got scared 😂

Reply:
But it does make sense to just render a plain `<p>`